experimental/virtio/src/lib.rs

```diff
@@ -55,3 +57,9 @@ pub trait Write {
 
 /// The vendor ID for virtio PCI devices.
 const PCI_VENDOR_ID: u16 = 0x1AF4;
+
+/// Memory address translation functions.
+pub trait Translator: Fn(VirtAddr) -> Option<PhysAddr> {}
```

Comment:
nit: the P->V transform was called "Translator" in the base PCI transport implementation (https://github.com/project-oak/oak/blob/main/third_party/rust-hypervisor-firmware-virtio/src/lib.rs#L18). I think this way around is better, so perhaps rename the trait in https://github.com/project-oak/oak/blob/main/third_party/rust-hypervisor-firmware-virtio/ for consistency.